projects/step-core/src/lib/services/dashboard.service.ts

```diff
@@ -5,6 +5,8 @@ import { ViewRegistryService } from './view-registry.service';
 import { downgradeInjectable, getAngularJSGlobal } from '@angular/upgrade/static';
 import { AJS_MODULE } from '../shared';
 
+const ONE_DAY_MS = 86400000;
```

Comment:
```suggestion
const ONE_DAY_MS = 24 * 60 * 60 * 1000;
```
I would write it like this.